src/components/results/ChartComponent.wc.svelte

```diff
@@ -625,6 +683,92 @@
             />
         </div>
     {/if}
+    {#if enableSorting}
+        <div part="lens-chart-sort-buttons">
+            <button
+                part="lens-chart-sort-button"
+                class:active={sortBy === "value"}
+                onclick={() => toggleSortBy("value")}
+                title="Sort by value"
+                aria-label="Sort by value {sortBy === 'value'
+                    ? sortOrder === 'asc'
+                        ? 'ascending'
+                        : 'descending'
+                    : ''}"
+            >
+                <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    height="24px"
+                    viewBox="0 -960 960 960"
+                    width="24px"
+                    fill="#e3e3e3"
+                    ><path
+                        d="M120-240v-80h240v80H120Zm0-200v-80h480v80H120Zm0-200v-80h720v80H120Z"
+                    /></svg
+                >
+                <svg
+                    width="20"
+                    height="20"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="2"
+                >
+                    {#if sortBy === "value" && sortOrder === "asc"}
+                        <!-- Arrow up for ascending -->
+                        <line x1="12" y1="19" x2="12" y2="5"></line>
+                        <polyline points="5 12 12 5 19 12"></polyline>
+                    {:else}
+                        <!-- Arrow down for descending (default) -->
```

Comment:
I would change the grayed out button to an up arrow because ascending is what you get when you click the button.